docs/features/recording.md

```diff
@@ -0,0 +1,137 @@
+
+# Room Recording (Beta)
+
+La Suite Meet offers a room recording feature that is currently in beta, with ongoing improvements planned.
+
+The feature allows users to record their room sessions. When a recording is complete, the room owner receives a notification with a link to download the recorded file. Recordings are automatically deleted after `RECORDING_EXPIRATION_DAYS`.
+
+**Current Limitations**:
+
+* Users cannot record and transcribe simultaneously.
+* Recording layout cannot be configured from the frontend. By default, the egress captures the active speaker and any shared screens.
+* Shareable links with an embedded video player are not yet supported.
+
+> [!NOTE]
+> Questions? Open an issue on [GitHub](https://github.com/suitenumerique/meet/issues/new?assignees=&labels=bug&template=Bug_report.md) or join our [Matrix community](https://matrix.to/#/#meet-official:matrix.org).
+
+
+## Special requirements
+
+To use the room recording feature, the following components are required:
+
+- A running LiveKit Egress server capable of handling room composite recordings.
+- A S3-compatible object storage that supports webhook events to notify the backend when recordings are uploaded.
+- An email service to notify room owners when a recording is available for download.
+- Webhook events configured between LiveKit Server and the backend.
```

Comment:
Shouldn't celery be required too? (I am not sure at all, I haven't found a celery task, just feeling that asynchronous tasks like these might require celery somewhere)